Signal/src/Models/TSMessageAdapaters/TSGenericAttachmentAdapter.m

```diff
@@ -132,10 +127,11 @@ - (UIView *)mediaView
         fileTypeLabel.text = fileExtension.uppercaseString;
         fileTypeLabel.textColor = textColor;
         fileTypeLabel.lineBreakMode = NSLineBreakByTruncatingTail;
-        fileTypeLabel.font = [self fileTypeLabelFont];
+        fileTypeLabel.font = [UIFont ows_mediumFontWithSize:20.f];
+        fileTypeLabel.adjustsFontSizeToFitWidth = YES;
         CGRect fileTypeLabelFrame = CGRectZero;
         fileTypeLabelFrame.size = [fileTypeLabel sizeThatFits:CGSizeZero];
-        fileTypeLabelFrame.size.width = floor(MIN(self.iconSize * 0.5f, fileTypeLabelFrame.size.width));
+        fileTypeLabelFrame.size.width = ceil(MIN(self.iconSize * 0.45f, fileTypeLabelFrame.size.width));
```

Comment:
Tweaked the appearance the arbitrary attachments. There's more to do here, but it's progress.